### ISSUE TITLE
Error building with Xcode 14.3

### ISSUE THREAD

**Issue by sgabriel** [software-mansion/react-native-svg#2038]
# Bug

## Unexpected behavior
When building a project using the latest version of this library, the following error occurs. 

```
In file included from <project-paht>/node_modules/react-native-svg/apple/Elements/RNSVGUse.mm:16:
<project-paht>/ios/Pods/Headers/Public/React-Fabric/react/renderer/components/view/conversions.h:111:39: error: 'value' is unavailable: introduced in iOS 12.0
          ? std::optional<Float>(base.value() * floatFromYogaFloat(value.value))
```

Other libraries have the same error and it seems to be related to Xcode 14.3 see [this PR](https://github.com/th3rdwave/react-native-safe-area-context/pull/370)

## Possible fix: 
Change this [line](https://github.com/software-mansion/react-native-svg/blob/main/RNSVG.podspec#L26) from 
```
s.platforms       = { ios: '11.0', tvos: '11.0' }
```
to
```
s.platforms       = { ios: '12.4', tvos: '11.0' }
```

**Comment by rarenatoe** [comment]
shameless bump

**Comment by RichardLindhout** [comment]
I want a release :)

**Comment by josmmv** [comment]
When you will release this?

**Comment by RichardLindhout** [comment]
I used patch-package in the meantime btw :)

**Comment by WoLewicki** [comment]
It's released in v13.10.0 🚀
